.github/workflows/tests.yml

```diff
@@ -0,0 +1,35 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - development
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment:
+      name: development
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v4.0.3
+      with:
+        node-version: 'lts/*'
+        cache: 'yarn'
+
+    - name: Install dependencies
+      run: yarn install
+
+    - name: Build project
+      run: CI=false yarn build
```

Comment:
I think this step actually needs to be in the deploy (this creates a built site to be deployed)

Reply:
You may also need the preceding step (yarn install) in that file as well. I believe it needs to be both places.

Reply:
Ahhhh, I am not familiar with react. That makes sense. I think I also have to install node first (based on some googling), so I've added that as well.

Reply:
Got that build section in the deploys and I've removed it from tests.